api/configuration/references.ts

```diff
@@ -28,7 +28,7 @@ export const SERVICES_CCD_DATA_STORE_API_PATH = 'services.ccd.dataApi'
 export const SERVICES_CCD_CASE_ASSIGNMENT_API_PATH = 'services.ccd.caseAssignmentApi'
 
 export const SERVICES_DOCUMENTS_API_PATH = 'services.documents.api'
-
+export const SERVICES_FEE_AND_PAY_API_PATH = 'services.feeAndPayApi'
```

Comment:
Just a question... why does this one have a different naming convention to the ones directly below it?

Reply:
i know, didnt want to change the naming convention just incase its used somewhere else